orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/RollingPushStage.java

```diff
@@ -56,6 +57,8 @@
             .withTask("checkForRemainingTerminations", CheckForRemainingTerminationsTask.class);
         }
       )
+      .withTask("cleanUpTags", CleanUpTagsTask.class)
+      .withTask("monitorTagCleanUp", MonitorKatoTask.class)
```

Comment:
Probably need to flag this based on whether entity tags is enabled. Take a look at https://github.com/spinnaker/orca/blob/master/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/CloneServerGroupStage.groovy for an example on how to do it...